### PR TITLE
feat(conference) implement start{Audio,Video} muted client side

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: react/features/base/conference/middleware.any.js [hunk 1]
```diff
@@ -220,6 +222,20 @@ function _conferenceJoined({ dispatch, getState }, next, action) {
         dispatch(openDisplayNamePrompt(undefined));
     }
 
+    const participantCont = getParticipantCount(state);
```
Comment (bgrozev) [inline react/features/base/conference/middleware.any.js:225]: Typo: cont (could have been worse)

Comment (saghul) [reply react/features/base/conference/middleware.any.js]: 👍

**Comment by damencho** [comment]
Interesting it break StartMutedTest test :)

**Comment by jallamsetty1** [comment]
@saghul why do we need this PR when startAudioMuted/startVideoMuted are already handled here - https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/conference/actions.js#L137
I verified that it seems to be working as expected.

**Comment by damencho** [comment]
The intention is to remove the start muted logic from jicofo. But that relies on the fact that all participants are reading config.js and can mute themselves, but this is not the case for jigasi for example, so I will close it for now ...

**Comment by damencho** [comment]
I had discussed that offline with Boris and with Saul.

**Comment by saghul** [comment]
Sounds good! Now we know.

**Pull request closed without merging**